imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -77,13 +63,17 @@ def initialize(self) -> None:
         self.mf6 = XmiWrapper(
             lib_path=self.dfm_metamod_config.kernels.modflow6.dll,
             lib_dependency=self.dfm_metamod_config.kernels.modflow6.dll_dep_dir,
-            working_directory=self.dfm_metamod_config.kernels.modflow6.work_dir,
+            working_directory=Path(
+                self.dfm_metamod_config.kernels.modflow6.work_dir.absolute()
```

Comment:
The `absolute` should not be necessary, we take already care for that in the `config` module by calling `resolve`

Reply:
not done. If I drop the ". absolute()", or if I also drop the path constructor and just use 
`working_directolry =self.dfm_metamod_config.kernels.modflow6.work_dir `
Then after the call to "self.dfm.initialize()" the modflow working directory is miraculously changed from XXX\MF6 to XXX\dflow_fm\MF6
This is maybe due to the fact that Python uses so much references. By assigning a deep copy of the path this effect is avoided.

Reply:
> By assigning a deep copy of the path this effect is avoided.

In that case I would make explicitly a deepcopy to make the intention clear.

Reply:
done. Now using 
`            working_directory=copy.deepcopy(
                self.dfm_metamod_config.kernels.modflow6.work_dir.absolute()
            )`
I tired dropping the .absolute() as well, but then I still had the problem outlined above. baffling.